x-pack/plugins/ml/public/application/components/model_snapshots/model_snapshots_table.tsx

```diff
@@ -0,0 +1,318 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useEffect, useState } from 'react';
+import { i18n } from '@kbn/i18n';
+
+// @ts-ignore
+import { formatDate } from '@elastic/eui/lib/services/format';
```

Comment:
Why don't you import from `@elastic/eui`?

Reply:
ah! this wasn't available in the past.

Reply:
Changed in ac22c8ce86162fd9d4db4687850de4ae33873f4b